src/quote/getQuoteAndBuyCoverInputs.ts

```diff
@@ -0,0 +1,161 @@
+import axios, { AxiosError } from 'axios';
+import dotenv from 'dotenv';
+
+import { calculatePremiumWithCommissionAndSlippage } from '../buyCover/calculatePremiumWithCommissionAndSlippage';
+import {
+  CoverAsset,
+  CoverId,
+  DEFAULT_COMMISSION_RATIO,
+  DEFAULT_SLIPPAGE,
+  MINIMUM_COVER_PERIOD,
+  NEXUS_MUTUAL_DAO_TREASURY_ADDRESS,
+  TARGET_PRICE_DENOMINATOR,
+} from '../constants/buyCover';
+import {
+  Address,
+  CoverRouterQuoteResponse,
+  ErrorApiResponse,
+  GetQuoteApiResponse,
+  GetQuoteResponse,
+  IntString,
+  Integer,
+  PoolCapacity,
+} from '../types';
+
+dotenv.config();
+
+type CoverRouterQuoteParams = {
```

Comment:
I think we should export all types, so consumers can import them when they want to type variables

Reply:
I try to export pretty much everything. Even internally it's sometimes annoying if you can't import a type when you want to type some params objects or something. I have been burned by libs a lot of times, looking at you wagmi 👀. But feel free to leave it like this